docs/index.md

```diff
@@ -1,37 +1,37 @@
 # Your Universal Cellular Automata
 
 <p align="center">
-<img src="https://raw.githubusercontent.com/riveSunder/yuca/master/assets/s11_slow_glider_vagopteron_glider.gif">
+<img src="https://raw.githubusercontent.com/riveSunder/yuca_docs/master/assets/s11_slow_glider_vagopteron_glider.gif">
   <br><br><strong>Left:</strong> Evolved slow glider from s11 Glaberish CA rule set. <strong>Right:</strong> evolved glider in <em>Vagopteron</em> Lenia rule set. 
 </p>
 
 ## A continuous cellular automata research library
 
 
-####  Davis, Q, Tyrell and Bongard, Josh. [Selecting Continuous Life-Like Cellular Automata for Halting Unpredictability: Evolving for Abiogenesis](https://rivesunder.github.io/yuca/evo_abio). Accepted to Proceedings of the Genetic and Evolutionary Computation Conference Companion (2022).  
+####  Davis, Q, Tyrell and Bongard, Josh. [Selecting Continuous Life-Like Cellular Automata for Halting Unpredictability: Evolving for Abiogenesis](https://rivesunder.github.io/yuca_docs/evo_abio). Accepted to Proceedings of the Genetic and Evolutionary Computation Conference Companion (2022).  
 
 * ArXiv -> [https://arxiv.org/abs/2204.07541](https://arxiv.org/abs/2204.07541)
-* [Blog post](https://rivesunder.github.io/yuca/evo_abio)
-* [Summary poster](https://raw.githubusercontent.com/riveSunder/yuca/gecco_2022_pages/assets/halting_evo/pos237s1.pdf)
-* [Notebook](https://github.com/riveSunder/yuca/blob/master/notebooks/halting_evo.ipynb) Try it on: [mybinder](https://mybinder.org/v2/gh/rivesunder/yuca/master?labpath=notebooks%2Fhalting_evo.ipynb) -> [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/rivesunder/yuca/master?labpath=notebooks%2Fhalting_evo.ipynb), [colab](https://colab.research.google.com/github/rivesunder/yuca/blob/master/notebooks/halting_evo.ipynb) -> [![Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rivesunder/yuca/blob/master/notebooks/halting_evo.ipynb) 
-* [Bokeh app glider demo -- watch evolved gliders](https://mybinder.org/v2/gh/riveSunder/yuca/gecco_2022_pages?urlpath=/proxy/5006/gecco_bokeh_app) Note that this runs on [mybinder.org](https://mybinder.org), a great free cloud notebook service that can be a little slow. 
+* [Blog post](https://rivesunder.github.io/yuca_docs/evo_abio)
```

Comment:
Let's go ahead and revert the links to the gh-pages to point to `https://rivesunder.github.io/yuca` instead of yuca_docs. This should decrease navigation confusion while still offloading the larger assets storage to yuca_docs.